src/actions/getPlayers.js

```diff
@@ -0,0 +1,23 @@
+import { getPlayersPromise } from './helpers/getPlayersHelper'
+
+const getPlayers = (rooms) => {
+  return (dispatch) => {
```

Comment:
https://github.com/duyoji/oni-on-cheese/pull/70#issuecomment-353726025

According to above error, getPlayers should return Promise object because you use [`then`](https://github.com/duyoji/oni-on-cheese/pull/70/files#diff-e504f79161efd34095c79ca089533da0R34) in your test code. 

But you return just function here. That's why console says `TypeError: Cannot read property 'then' of undefined`


I think you can write code like following.

```js
const getPlayers = (rooms) => {
    return async (dispatch) => {...}
}
```

or

```js
const getPlayers = (rooms) => {
    return Promise.all(...)
}

```